.github/main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x]
```

Comment:
no 9.0?

Reply:
I doubt anyone is using it? We should support 12 & 14